editor/src/components/navigator/left-pane.tsx

```diff
@@ -763,6 +768,249 @@ const GithubSpinner = () => {
   )
 }
 
+interface RepositoryRowProps extends RepositoryEntry {
+  importPermitted: boolean
+}
+
+const RepositoryRow = (props: RepositoryRowProps) => {
+  const colorTheme = useColorTheme()
+
+  const dispatch = useEditorState((store) => store.dispatch, 'RepositoryRow dispatch')
+
+  const projectID = useEditorState((store) => store.editor.id, 'RepositoryRow projectID')
+
+  const importRepository = React.useCallback(() => {
+    const parsedTargetRepository = parseGithubProjectString(props.fullName)
+    if (parsedTargetRepository == null) {
+      dispatch(
+        [
+          showToast(
+            notice(
+              `Error when attempting to import a repository with repo: ${props.fullName}`,
+              'ERROR',
+            ),
+          ),
+        ],
+        'everyone',
+      )
+    } else {
+      void getBranchContent(
+        dispatch,
+        parsedTargetRepository,
+        forceNotNull('Should have a project ID.', projectID),
+        props.defaultBranch,
+      )
+    }
+  }, [dispatch, projectID, props.fullName, props.defaultBranch])
+
+  return (
+    <div
+      style={{
+        minHeight: 40,
+        display: 'flex',
+        flexDirection: 'row',
+        gap: 8,
+        alignItems: 'center',
+        paddingLeft: 8,
+        paddingRight: 8,
+        paddingBottom: 8,
+        paddingTop: 8,
+        borderBottom: '1px solid #ccc',
+      }}
+    >
+      <div
+        style={{
+          borderRadius: '50%',
+          width: 20,
+          height: 20,
+          border: '1px solid #ccc',
+          backgroundImage: `url(${props.avatarUrl})`,
+          backgroundPosition: 'center',
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+        }}
+      />
+      <div style={{ flexGrow: 1 }}>
+        <span style={{ fontWeight: 600, textOverflow: 'ellipsis' }}>
+          {props.name ?? props.fullName}
+        </span>{' '}
+        <br />
+        <span style={{ opacity: 0.5 }}>
+          {props.private ? 'private' : 'public'} · <TimeAgo date={props.updatedAt} />
+        </span>
+      </div>
+      <button
+        style={{
+          fontSize: 11,
+          background: colorTheme.buttonBackground.value,
+          boxShadow: 'none',
+          border: 'none',
+          height: 22,
+          color: colorTheme.inlineButtonColor.value,
+          borderRadius: 2,
+        }}
+        onMouseUp={importRepository}
+      >
+        Import
```

Comment:
for whatever reason beyond me, this does not use `cursor: pointer` when hovered 🧐

Reply:
I'm thinking we might have a global override for that somewhere, meaning `button` by default does not use the pointer cursor. It's one of those "application / editor UX" vs "web UX" clashes.

Reply:
Fixed in 1a05bc536.